### PR TITLE
Dedicated circle name for groups

### PR DESCRIPTION
**Pull request by annando**
There is now a separate circle for groups. When following a lot of groups, it can help to have them separated.

### CODE REVIEW

**Review by MrPetovan — COMMENTED** [review]
Isn't it something that would be better solved with a meta-circle like Followers?

**Comment by annando** [comment]
A circle like "Followers" is great when you address a posting to that circle. But here it is meant for selecting all postings that are done in a group.

File: src/Model/User.php [hunk 1]
```diff
@@ -1208,6 +1226,11 @@ public static function create(array $data): array
 
 		DBA::update('user', $fields, ['uid' => $uid]);
 
+		$def_gid_groups = Circle::create($uid, DI::l10n()->t('Groups'));
+		if (!$def_gid_groups) {
```
Comment (MrPetovan) [inline src/Model/User.php]: Are you sure about this condition?

Comment (annando) [reply src/Model/User.php]: Your comment is a very polite "this is wrong" :-)

Comment (MrPetovan) [reply src/Model/User.php]: It looked wrong to me but I didn't want to assume!

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**